ui/css/design-system/_colors.scss

```diff
@@ -24,13 +24,11 @@ $color-map: (
   'warning-muted': --color-warning-muted,
   'warning-inverse': --color-warning-inverse,
   'success-default': --color-success-default,
-  'success-alternative': --color-success-alternative,
   'success-muted': --color-success-muted,
   'success-inverse': --color-success-inverse,
   'info-default': --color-info-default,
   'info-muted': --color-info-muted,
   'info-inverse': --color-info-inverse,
-  'info-disabled': --color-info-disabled,
```

Comment:
Must have been a bad merge `info-disabled` no longer exists in the codebase

![Screenshot 2024-06-07 at 4 06 04 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/b318ff68-618d-403b-8d28-d2a01accb2f4)